pkg/controller/atlasproject/team_reconciler_test.go

```diff
@@ -3,24 +3,30 @@ package atlasproject
 import (
 	"context"
 	"errors"
+	"net/http"
 	"testing"
 
+	"github.com/stretchr/testify/mock"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/atlas"
+
 	"github.com/stretchr/testify/assert"
 	"go.mongodb.org/atlas/mongodbatlas"
 
-	atlasmock "github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/mocks/atlas"
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+	"go.mongodb.org/atlas-sdk/v20231115008/mockadmin"
```

Comment:
please fix package ordering by grouping:
1. golang SDK packages
2. external packages
3. internal packages

in separate blocks